apps/landing/src/layouts/Layout.astro

```diff
@@ -0,0 +1,73 @@
+---
+// src/layouts/Layout.astro
```

Comment:
Remove comments

Reply:
Without these delimiters(---), the file will not be able to process the code correctly.